IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -232,7 +317,45 @@ def describe_screen(screen_id):
     return plate_results_df
 
 
-# In[10]:
+# In[22]:
+
+
+def collect_metadata(idr_name, values_list):
+    """Data collection and saving pipeline"""
+    data_dir = pathlib.Path("../data/metadata")
+    if data_dir.exists() == False:
+        os.mkdir(data_dir)
```

Comment:
Please keep directory functionality within `pathlib`. See [`pathlib.Path.mkdir()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir).

Also note the function argument `exist_ok` :) (this will make line 326 and 327 unnecessary)

Reply:
Is there a specific reason for only using `pathlib`? I changed uses of `os` to `pathlib`.

Added in ade7519d00aa36e6d4211189cdff064fdecc89b7.

Reply:
pathlib has replaced os for most things. It seems to me to be easier and more robust to errors.

https://towardsdatascience.com/dont-use-python-os-library-any-more-when-pathlib-can-do-141fefb6bdb5